nifi-commons/nifi-security-utils/pom.xml

```diff
@@ -68,6 +68,9 @@
                 <configuration>
                     <excludes combine.children="append">
                         <exclude>src/test/resources/xxe_template.xml</exclude>
+                        <!-- This file is copied from https://github.com/jeremyh/jBCrypt
+because the binary is compiled for Java 8 and we must support Java 7 -->
```

Comment:
@alopresto we no longer need to support Java 7.  We can remove this.  Not saying this for this review but is worth resolving later.

Reply:
Yeah I had the same thought but did not want to delay this PR any more. Will open a Jira for this.